README.md

```diff
@@ -0,0 +1,303 @@
+# frontend-build
+
+## Warning: Package Moved
```

Comment:
@abdullahwaheed alerted me to the possibility of having the warning be more "dangerous" looking, such as [here](https://www.npmjs.com/package/babel-plugin-react-intl):

![image](https://github.com/openedx/frontend-build/assets/759355/2c008258-2dad-4943-a1d9-119e6043560b)

Not sure how to do that yet, but we should probably look into it.  We can do that later, though, this is not a blocker for this PR.

Reply:
Looks like it's not too bad and I can go back and do that for the packages we've already deprecated: https://docs.npmjs.com/deprecating-and-undeprecating-packages-or-package-versions